lib/identity_cache/configuration_dsl.rb

```diff
@@ -262,6 +265,15 @@ def attribute_dynamic_fetcher(attribute, fields, values, unique_index) #:nodoc:
         end
       end
 
+      def transform_case_insensitive_values(fields, values, case_insensitive_fields)
+        return values if case_insensitive_fields.empty?
+
+        fields.zip(values).map do |field, value|
+          next(value.downcase) if case_insensitive_fields.include?(field)
```

Comment:
What's the collation on the fields you are trying to index? I think the comment you added to the README is very misleading.